sdk/keyvault/azure-security-keyvault-keys/inc/azure/keyvault/keys/recover_deleted_key_operation.hpp

```diff
@@ -68,13 +68,13 @@ namespace Azure { namespace Security { namespace KeyVault { namespace Keys {
      * Since C++ doesn't offer `internal` access, we use friends-only instead.
      */
     RecoverDeletedKeyOperation(
-        std::shared_ptr<Azure::Security::KeyVault::_internal::KeyVaultPipeline> keyvaultPipeline,
+        std::shared_ptr<Azure::Security::KeyVault::Keys::KeyClient> keyClient,
         Azure::Response<Azure::Security::KeyVault::Keys::KeyVaultKey> response);
 
     RecoverDeletedKeyOperation(
-        std::shared_ptr<Azure::Security::KeyVault::_internal::KeyVaultPipeline> keyvaultPipeline,
+        std::shared_ptr<Azure::Security::KeyVault::Keys::KeyClient> keyClient,
```

Comment:
nit: I know these are private, but to avoid confusion when folks copy/paste the pattern in the future, keep the param order consistent and swap them here.